__tests__/app.test.js

```diff
@@ -18,65 +18,26 @@ afterAll(() => {
   return db.end();
 });
 
-describe("GET /api/articles/:article_id", () => {
-  it("should return the required keys with the correct value types", () => {
-    return request(app)
-      .get("/api/articles/1")
-      .expect(200)
-      .then((response) => {
-        expect(response.body.article.length).toBe(1);
-        response.body.article.forEach((article) => {
-          expect(article.article_id).toBe(1);
-          expect(typeof article.title).toBe("string");
-          expect(typeof article.topic).toBe("string");
-          expect(typeof article.author).toBe("string");
-          expect(typeof article.body).toBe("string");
-          expect(typeof article.created_at).toBe("string");
-          expect(typeof article.votes).toBe("number");
-          expect(typeof article.article_img_url).toBe("string");
-        });
-      });
-  });
-  it("should return 400 if given an ID that is not a number", () => {
-    return request(app)
-      .get("/api/articles/dog")
-      .expect(400)
-      .then((response) => {
-        expect(response.body).toEqual({msg: "Invalid Input!"});
-      });
+describe('GET /api', () => {
+  it('should return a JSON object', () => {
+    return request(app).get('/api')
+    .expect(200)
+    .then((response) => {
+      expect(JSON.parse(response.text)).toBeInstanceOf(Object)
+    })
+    
   });
-  it('should return 404 if given a number but unable to find it', () => {
-    return request(app)
-    .get("/api/articles/9999")
-    .expect(404)
+  it('should return the same data as the original JSON file', () => {
+    return request(app).get('/api')
+    .expect(200)
     .then((response) => {
-      expect(response.body).toEqual({err: "No user found for user_id: 9999!"});
-    });
+      const parsedEndpoints = JSON.parse(response.text)
+      expect(parsedEndpoints).toEqual(JSONendpoints)
+    })
     
   });
 });
 
-describe('GET /api', () => {
-    it('should return a JSON object', () => {
-        return request(app).get('/api')
-        .expect(200)
-        .then((response) => {
-            expect(JSON.parse(response.text)).toBeInstanceOf(Object)
-        })
-        
-    });
-    it('should return the same data as the original JSON file', () => {
-        return request(app).get('/api')
-        .expect(200)
-        .then((response) => {
-            const parsedEndpoints = JSON.parse(response.text)
-            expect(parsedEndpoints).toEqual(JSONendpoints)
-        })
-        
-    });
-});
-
-
 describe('GET /api/topics', () => {
     it('should return the required keys with the correct value types', () => {
         return request(app).get('/api/topics')
```

Comment:
Asserting the property exists seems unnecessary if we're checking the type of the key value in the following assertions.